<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>razrab</class>
 <widget class="QWidget" name="razrab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twRazrab">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectAll">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Показать всех разработчиков</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Окно отладки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>701</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Разработчики</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>291</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="10" column="0">
     <widget class="QLineEdit" name="leComandRazrab"/>
    </item>
    <item row="8" column="0">
     <widget class="QLineEdit" name="lePostRazrab"/>
    </item>
    <item row="12" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnUpdate">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="lePhoneRazrab"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Должность</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QRadioButton" name="radOffice">
      <property name="text">
       <string>Работа в офисе</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Номер телефона</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="leFIORazrab"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>ФИО</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="leEmailRazrab"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Отдел</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>На главную</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teRazrab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>681</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leRazrabNameSearch">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string extracomment="Введите ФИО Ученика для поиска"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
